test/libponyc/sugar.cc

```diff
@@ -553,31 +553,115 @@ TEST_F(SugarTest, FunctionNoReturnBody)
 }
 
 
-TEST_F(SugarTest, FunctionParamMustBeId)
+TEST_F(SugarTest, MethodParamMustBeId)
```

Comment:
because we check parameter names after sugaring `be`, `fun`, and `new`, i updated the tests to check for each type as it would otherwise be possible to break for say `be` because we were only checking `fun`.

accordingly, the test name was updated.